src/templates/settings/application.html

```diff
@@ -58,6 +58,14 @@
         <input type="text" class="textfield  textfield-small  max-fps-input" autocomplete="off" name="max-fps"/>
       </div>
 
+      <div class="settings-item">
+        <label for="color-format">Color format</label>
+        <select id="color-format" class="color-format-select">
```

Comment:
Small UI nit: in settings-application.css, there is a CSS rule applied to `grid-width-select`. 
We want the same rule to be applied to this element for consistency.

https://github.com/juliandescottes/piskel/blob/master/src/css/settings-application.css#L57-L59